### ISSUE TITLE
Specifying queues at launch

### ISSUE THREAD

**Issue by j-mcnally** [kmullin/resque-sliders#7]
I am working on adding support for a yml to be passed KEWatcher that would set default queue / worker ratios at boot time.

I am deploying to amazon-ebs so this is important as server may come and go as scaling happens.

I think it also makes sense to use resque-retry in my scenario so if a worker is killed off it would get re-tried with a different worker.

Thoughts?

**Comment by kmullin** [comment]
Hey,

I think predefining the queue names via a config file is a great idea. I welcome the PR for that! :smile:

resque-sliders is not aimed at monitoring the status of failed jobs, so you might want to use another plugin like resque-retry to enable that functionality.

**Comment by j-mcnally** [comment]
Yes i have the config stuff working in my fork but things got messy in my
pr  so i closed it basically i want to rebase that feature off ur master
and will send a new PR. Yes i am using resque retry for the other stuff and
love it

Sent from my iPhone

On Sep 3, 2013, at 12:56 PM, Kevin Mullin notifications@github.com wrote:

Hey,

I think predefining the queue names via a config file is a great idea. I
welcome the PR for that! [image: :smile:]

resque-sliders is not aimed at monitoring the status of failed jobs, so you
might want to use another plugin like resque-retry to enable that
functionality.

—
Reply to this email directly or view it on
GitHubhttps://github.com/kmullin/resque-sliders/issues/7#issuecomment-23733223
.

**Comment by vazzzabi** [comment]
So, can you help me? How i can add queues at launch automatically? And the second - how i can add nodes (another hosts/servers)? At screenshots i saw 3 node, how you configure it?
